content/src/roadmaps/license-tasks/landlord-registration.md

```diff
@@ -6,12 +6,15 @@ urlSlug: landlord-registration
 name: Register as a Landlord
 webflowName: "Residential Landlord: Landlord Registration"
 summaryDescriptionMd: Before renting out a **one- or two-unit residential
-  property**, you must register as a landlord with your municipal clerk. You
+  property**, you must register as a landlord with your local clerk. You
   need to register each time you rent out a new property. You also need to give
   your tenant(s) a copy of the completed application form.
 industryId: residential-landlord
 webflowId: 66f3198052d588875cacb94c
 licenseCertificationClassification: undefined
+requiresLocation: true
+callToActionLink: ${municipalityWebsite}
+callToActionText: Visit the ${municipality} Website
```

Comment:
[dust] Idk how hard this is, but I think having this variable be `municipalityName` instead might be more descriptive and less ambiguous.

Reply:
This seems doable and makes sense, I'll see if I can make that change in this PR